### PR TITLE
Update palo-alto-networks-cortex-xdr---investigation-and-response.md

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Status
Ready

## Related Issues
[fixes: link to the issue](https://github.com/demisto/content/pull/15356)

## Description
Added info about Cortex XDR AWS IAM User Access Investigation

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md [hunk 1]
```diff
@@ -62,6 +64,40 @@ After the remediation, if there are no new alerts, the playbook stops the alert
...
+
+Then the playbook enriches information about the user who connected to the environment through the AWS IAM integration using the [AWS IAM - User enrichment](https://xsoar.pan.dev/docs/reference/playbooks/aws-iam---user-enrichment) sub-playbook. The sub-playbook lists the user access keys and retrieves information about the IAM user, including the user's creation date, path, unique ID, and ARN.  From this, it can be seen if these user keys are active and the analyst can block these keys later in the investigation if they are causing malicious activities.
+
+
+Then the playbook validates that the access key type is AKIA (which marks this as a user key). If the access key is AKIA, queries are run to retrieve the last 100 API calls made with the access key and retrieve actions performed by the user in the last 7 days. This information shows who made the call, and provides information about the IP address and data about which user was used in the request, what operation was performed, the status of the operation and on what resource it was executed.
+
+Not the investigation starts.
```
Comment (altmannyarden) [inline content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md:86]: "Not" - should be "Now"?

File: content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md [hunk 2]
```diff
@@ -62,6 +64,40 @@ After the remediation, if there are no new alerts, the playbook stops the alert
...
+First the playbook checks if there were new IP addresses that were found on the XQL queries that did not appear in the original alert and enriches them.
+Then the analyst manually reviews the results of the XQL queries from the previous steps to determine if this is a true positive event. The analyst investigates the operation performed by the access key and the user. The analyst examines the executed operations, by who it was executed, on which resource, and the operation status.
```
Comment (altmannyarden) [inline content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md:88]: Then the analyst manually reviews the results of the XQL queries from the previous steps to determine if this is a true positive event. The analyst investigates the **operation** performed by the access key and the user.
Need to be operations?

Comment (ShirleyDenkberg) [reply content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md]: Done.

File: content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md (hunk 1 above)
Comment (ShirleyDenkberg) [reply content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md]: done.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**